src/Microsoft.PowerShell.ConsoleGuiTools/ConsoleGui.cs

```diff
@@ -118,14 +124,37 @@ public void Start(ApplicationData applicationData)
             var list = new ListView(items)
             {
                 X = 3,
-                Y = 3,
+                Y = 1,
                 Width = Dim.Fill(2),
-                Height = Dim.Fill(2),
+                Height = Dim.Fill(0),
                 AllowsMarking = applicationData.PassThru
             };
             
             win.Add(list);
 
+            win.Enter_Pressed += () =>
+            {
+                // Regardless of whether the item was already marked, if the user presses
+                // ENTER, mark that item. This feels the most intuitive. The alternatives are:
+                // (a) Assume the user means to choose the item that's selected and mark it.
+                // (b) Act like ESC except Pass Through an marked items.
+                // (c) Toggle the selected item.
+                // Option (c) violates the principle of least astonishment: If the user has an item marked, and selected
+                // and hits ENTER s/he would be surprised to see it not returned. Esp in the single choice (most common) case.
+                // Option (b) means in the most common case (a single item is wanted) ENTER does nothing. Again surprising.
+                // It seems (a) is the most intuitive so that's what's implemented.
+                list.Source.SetMark(list.SelectedItem, true);
+                Application.RequestStop();
```

Comment:
Since \<Enter> action is normally stop and close I'd do not assign other actions for \<Enter>. I'd expect that \<Space> action is "Select".